docker/images/ubuntu/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM ubuntu:18.04
 LABEL maintainer="Postman Labs <help@postman.com>"
 
-ARG NODE_VERSION=10
+ARG NODE_VERSION=16
```

Comment:
[Current LTS](https://nodejs.org/en) is `20`, but Nodesource only [supports up to `16` on ubuntu:18.04](https://github.com/nodesource/distributions?tab=readme-ov-file#ubuntu-versions).

Reply:
Note that this PR is about a year old. I no longer have a need for these changes and will therefore not work on this PR anymore. Feel free to pick up on this state.